sdk/monitor/Azure.Monitor.Ingestion/CHANGELOG.md

```diff
@@ -21,6 +21,10 @@
 - Removed Model `UploadLogsError` representing the error and the associated logs that failed when uploading a subset of logs to Azure Monitor.
 - Removed Model `UploadLogsStatus` indicating the status of a logs upload operation.
 
+## 1.1.0 (2022-10-11)
```

Comment:
Fix the date of release and this should be at the top of the changelog as this is the most recent release.

Reply:
updated!